### DISCUSSION TITLE
第2节课作业(20班)

### DISCUSSION THREAD

**Discussion by Farewell-CK** [InternLM/Tutorial#220] [作业]
宝子们，作业提交到这里哦

**Comment by deyouzuishuai** [comment]
https://zhuanlan.zhihu.com/p/677256830

**Comment by TtKkD** [comment]
http://t.csdnimg.cn/CYpeY

**Comment by LiyanJin** [comment]
https://blog.csdn.net/weixin_53443275/article/details/135563411

**Comment by TLzh** [comment]
https://blog.csdn.net/zhiho14/article/details/135541245

补个作业，还在审核中，但是应该就这个链接了，这次作业还是很有意思的。

**Comment by Walshyu** [comment]
https://github.com/Liltomato1998/hello-world/blob/main/LLM/%E7%AC%AC%E4%BA%8C%E8%8A%82%E8%AF%BE%E4%BD%9C%E4%B8%9A.md

**Comment by zhangyiyuan1112** [comment]
https://iqxyis6ody.feishu.cn/docx/S22bdAFPnoLoX7xp6dhcyWGfnYg?from=from_copylink

**Comment by qiaoxiaopang** [comment]
https://blog.csdn.net/weixin_74322438/article/details/135575057

**Comment by 20300240011** [comment]
https://zhuanlan.zhihu.com/p/677694241

**Comment by forestemperor** [comment]
https://github.com/forestemperor/InternLM_studynote/tree/main

**Comment by forestemperor** [comment]
https://github.com/forestemperor/InternLM_studynote/blob/main/stydynote/lesson2.md

**Comment by pooh0315** [comment]
https://blog.csdn.net/m0_74206611/article/details/135586108

**Comment by xiaocaipi** [comment]
https://github.com/xiaocaipi/InterLm-homework

**Comment by yalunren** [comment]
作业
![屏幕截图 2024-01-21 161135](https://github.com/InternLM/tutorial/assets/23251243/0ce33198-47ba-4491-a9b0-c6f46f4b0b39)
![屏幕截图 2024-01-21 184822](https://github.com/InternLM/tutorial/assets/23251243/74cebdde-3277-41d6-a53f-bdf87e9e1d29)
